-- Using CATKIN_DEVEL_PREFIX: /home/takrop/ros2_ws/build/hls_lfcd_lds_driver/devel
-- Using CMAKE_PREFIX_PATH: /home/takrop/catkin_ws/devel;/opt/ros/noetic
-- This workspace overlays: /home/takrop/catkin_ws/devel;/opt/ros/noetic
-- Override CMake install command with custom implementation using symlinks instead of copying resources
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.8.10", minimum required is "3") 
-- Using PYTHON_EXECUTABLE: /usr/bin/python3
-- Using Debian Python package layout
-- Using empy: /usr/lib/python3/dist-packages/em.py
-- Using CATKIN_ENABLE_TESTING: ON
-- Call enable_testing()
-- Using CATKIN_TEST_RESULTS_DIR: /home/takrop/ros2_ws/build/hls_lfcd_lds_driver/test_results
-- Forcing gtest/gmock from source, though one was otherwise available.
-- Found gtest sources under '/usr/src/googletest': gtests will be built
-- Found gmock sources under '/usr/src/googletest': gmock will be built
-- Found PythonInterp: /usr/bin/python3 (found version "3.8.10") 
-- Using Python nosetests: /usr/bin/nosetests3
-- catkin 0.8.10
-- BUILD_SHARED_LIBS is on
Traceback (most recent call last):
  File "/home/takrop/ros2_ws/build/hls_lfcd_lds_driver/catkin_generated/generate_cached_setup.py", line 22, in <module>
    code = generate_environment_script('/home/takrop/ros2_ws/build/hls_lfcd_lds_driver/devel/env.sh')
  File "/opt/ros/noetic/lib/python3/dist-packages/catkin/environment_cache.py", line 63, in generate_environment_script
    env_after = ast.literal_eval(output.decode('utf8'))
  File "/usr/lib/python3.8/ast.py", line 59, in literal_eval
    node_or_string = parse(node_or_string, mode='eval')
  File "/usr/lib/python3.8/ast.py", line 47, in parse
    return compile(source, filename, mode, flags,
  File "<unknown>", line 1
    ROS_DISTRO was set to 'foxy' before. Please make sure that the environment does not mix paths from different distributions.
               ^
SyntaxError: invalid syntax
CMake Error at /opt/ros/noetic/share/catkin/cmake/safe_execute_process.cmake:11 (message):
  execute_process(/usr/bin/python3
  "/home/takrop/ros2_ws/build/hls_lfcd_lds_driver/catkin_generated/generate_cached_setup.py")
  returned error code 1
Call Stack (most recent call first):
  /opt/ros/noetic/share/catkin/cmake/all.cmake:208 (safe_execute_process)
  /opt/ros/noetic/share/catkin/cmake/catkinConfig.cmake:20 (include)
  CMakeLists.txt:10 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/takrop/ros2_ws/build/hls_lfcd_lds_driver/CMakeFiles/CMakeOutput.log".
See also "/home/takrop/ros2_ws/build/hls_lfcd_lds_driver/CMakeFiles/CMakeError.log".
